tests/schema_change_test.cc

```diff
@@ -715,3 +715,24 @@ SEASTAR_TEST_CASE(test_schema_digest_does_not_change_with_functions) {
             e.execute_cql("create aggregate my_agg(int) sfunc my_add stype int finalfunc twice;").get();
         });
```

Comment:
Please document in the change log what cases you are testing. This will allow a reviewer to consider whether there are missing tests, and whether the tests do in fact test what the commit claims they test.

Reply:
I added commit message.